openmetadata-service/src/main/java/org/openmetadata/service/resources/teams/TeamResource.java

```diff
@@ -510,13 +513,133 @@ public CsvImportResult importCsv(
     return importCsvInternal(securityContext, name, csv, dryRun);
   }
 
+  @GET
```

Comment:
Let's not add this as another API end point. we will do this as part of another APIs